src/jpgimage.cpp

```diff
@@ -49,14 +51,40 @@ constexpr byte rst1_ = 0xd0;  //!< JPEG Restart 0 Marker (from 0xD0 to 0xD7 ther
 
 // Start of Frame markers, nondifferential Huffman-coding frames
 constexpr byte sof0_ = 0xc0;  //!< JPEG Start-Of-Frame marker
+constexpr byte sof1_ = 0xc1;  //!< JPEG Start-Of-Frame marker
+constexpr byte sof2_ = 0xc2;  //!< JPEG Start-Of-Frame marker
 constexpr byte sof3_ = 0xc3;  //!< JPEG Start-Of-Frame marker
 
 // Start of Frame markers, differential Huffman-coding frames
 constexpr byte sof5_ = 0xc5;  //!< JPEG Start-Of-Frame marker
+constexpr byte sof6_ = 0xc6;  //!< JPEG Start-Of-Frame marker
+constexpr byte sof7_ = 0xc6;  //!< JPEG Start-Of-Frame marker
 
 // Start of Frame markers, differential arithmetic-coding frames
+constexpr byte sof9_ = 0xc9;   //!< JPEG Start-Of-Frame marker
+constexpr byte sof10_ = 0xca;  //!< JPEG Start-Of-Frame marker
+constexpr byte sof11_ = 0xcb;  //!< JPEG Start-Of-Frame marker
+constexpr byte sof13_ = 0xcd;  //!< JPEG Start-Of-Frame marker
+constexpr byte sof14_ = 0xce;  //!< JPEG Start-Of-Frame marker
 constexpr byte sof15_ = 0xcf;  //!< JPEG Start-Of-Frame marker
 
+// JPEG process SOF markers
+constexpr Internal::TagDetails jpegProcessMarkerTags[] = {
```

Comment:
can't say I like this. In other places in this codebase, operator== is implemented and Exiv2::find is used.